tests/test_flows.py

```diff
@@ -52,6 +61,11 @@
 from prefect.utilities.hashing import file_hash
 
 
+@flow
+def test_flow():
+    pass
+
+
```

Comment:
It looks like this isn't necessary? The test functions below define their own `test_flow` in the test bodies so I _think_ those take precedence.

Reply:
Good catch! I actually want to keep this one, so I removed the other `test_flow` fixtures.